lib/util/stylesheet.js

```diff
@@ -101,9 +101,16 @@ const astHelpers = {
       node &&
       node.type === 'CallExpression' &&
       node.callee &&
-      node.callee.object &&
-      node.callee.object.name &&
-      objectNames.includes(node.callee.object.name)
+      node.callee.object && ((
+        node.callee.object.type === 'Identifier' &&
+        node.callee.object.name &&
+        objectNames.includes(node.callee.object.name)
+      ) || (
+        node.callee.object.type === 'MemberExpression' &&
+        node.callee.object.object.type === 'Identifier' &&
+        astHelpers.getImportSource(node.callee.object.object) === 'react-native' &&
```

Comment:
This can potentially be heavy operation. I wonder if it should be cached in some way. Also, maybe it should be moved as the last check?

Reply:
Did bump into some performance issues with unnecessary import checks. Decided to move it as the last check to improve performance.